generator/postcode_points_builder.cpp

```diff
@@ -84,77 +86,53 @@ void GetPostcodes(std::string const & filename, storage::CountryId const & count
     if (std::find(countries.begin(), countries.end(), countryId) == countries.end())
       continue;
 
-    auto postcode = fields[fieldIndices.m_postcodeIndex];
-    if (!transformPostcode(postcode))
-      continue;
-
-    CHECK_EQUAL(valueMapping.size(), index, ());
+    keyValuePairs.emplace_back(search::NormalizeAndSimplifyString(fields[fieldIndices.m_postcodeIndex]),
```

Comment:
Can you capitalize all the letters in postcodes? Lowercase postcodes look strange:
<img width="300px" src="https://github.com/organicmaps/organicmaps/assets/26939824/f3a43f0e-3385-462a-9ae2-0f9342a8a200"/>

i can't find any postcode formats where lowercase letters are used: https://en.wikipedia.org/wiki/List_of_postal_codes


(thank you so much for working on this, i think it'll make a lot of people happy!)